backend/src/main/java/backend/mulkkam/intake/domain/IntakeHistory.java

```diff
@@ -30,25 +30,19 @@ public class IntakeHistory {
     private Member member;
 
     @Column(nullable = false)
-    private LocalDateTime dateTime;
-
-    @Embedded
-    @AttributeOverride(name = "value", column = @Column(name = "intakeAmount", nullable = false))
-    private Amount intakeAmount;
+    private LocalDate historyDate;
 
     @Embedded
     @AttributeOverride(name = "value", column = @Column(name = "targetAmount", nullable = false))
```

Comment:
```suggestion
    @AttributeOverride(name = "value", column = @Column(name = "target_amount", nullable = false))
```

여기도 스네이크 형식으로 써야할 것 같아요!